### PR TITLE
Admin GUI via haproxy

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Use haproxy to server the admin gui

# Impacts
Admin GUI

# NEW Package(s) required
perl(Catalyst::Plugin::SmartURI)
perl(URI::SmartURI)

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* Haproxy server the admin gui, httpd.admin disabled by default

### CODE REVIEW

**Comment by fdurand** [comment]
Ready for review.

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/CertificatesView.vue [hunk 1]
```diff
@@ -104,6 +105,7 @@
               <p>
                 {{ $t('Some services must be restarted to load the new certificate.') }}
                 <span v-if="id === 'http'" v-html="$t('The {service} service needs to be restarted from the command-line.', { service: $strong('httpd.admin') })"></span>
```
Comment (julsemaan) [inline html/pfappserver/root/static.alt/src/views/Configuration/_components/CertificatesView.vue]: This isn't true anymore I believe

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/CertificatesView.vue [hunk 2]
```diff
@@ -26,6 +26,7 @@
...
```
Comment (julsemaan) [inline html/pfappserver/root/static.alt/src/views/Configuration/_components/CertificatesView.vue]: This isn't true anymore I believe

**Comment by julsemaan** [comment]
Also one thing I don't see is adding the new admin services in the protected services in the Services section of the admin. And we could also remove the protection of httpd.admin

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/CertificatesView.vue (hunk 1 above)
Comment (fdurand) [reply html/pfappserver/root/static.alt/src/views/Configuration/_components/CertificatesView.vue]: yes you are right

**Pull request merged**